MOONCLOCK/apps.py

```diff
@@ -235,26 +247,85 @@ def update(self, first, remaining_duration):
         hour_fee = str(fees['hourFee'])
         
         print('Recommended fees:', str(fees))
-
-        fee_string = str_cjust('{}{}:{}{}'.format(font.CHAR_SATOSHI, fastest_fee, font.CHAR_SATOSHI, hour_fee), 7)
+        
         if len(fastest_fee) + len(hour_fee) > 5:
-            self.display_group.clear()
-            self.display_group.render_string(fee_string, center=False)
+            fee_string = '{}{}:{}{}'.format(font.CHAR_SATOSHI, fastest_fee, font.CHAR_SATOSHI, hour_fee)
+            str_to_render = str_align(fee_string, 10, ' ', self.align)
         else:
-            self.display_group.clear()
-            self.display_group.render_string('{0} {1}'.format(font.CHAR_MONEY_BAG, fee_string), center=False)
-        self.display_group.show()  
+            fee_string = '{}{}:{}{}'.format(font.CHAR_SATOSHI, fastest_fee, font.CHAR_SATOSHI, hour_fee)
+            str_to_render = '{} {}'.format(font.CHAR_MONEY_BAG, str_align(fee_string, 7, ' ', self.align))
+        
+        self.display_group.clear()
+        self.display_group.render_string(str_to_render, center=False)
+        self.display_group.show()
 
 
 class Text(App):
-    def __init__(self, *args, text='', **kwargs):
+    def __init__(self, *args, text='', align='center', **kwargs):
         super().__init__(*args, **kwargs)
         self.text = text
+        self.align = align
 
     def update(self, first, remaining_duration):
         text = str(self.text)
         print('Show text: ', text)
 
         self.display_group.clear()
-        self.display_group.render_string(text, center=True)
-        self.display_group.show()  
+        self.display_group.render_string(str_align(text, 10, ' ', self.align), center=False)
+        self.display_group.show()
+
+class MarketCap(App):
```

Comment:
Something to be solved globally probably... not ideal

Reply:
✅